deployment/cloud-run/test_swagger_debug_detailed.py

```
@@ -25,17 +25,28 @@
-    
+
     server_thread = threading.Thread(target=run_server, daemon=True)
     server_thread.start()
-    
-    # Wait for server to start
+
+    # Wait for server to be ready with polling
     print("🔄 Starting server...")
-    time.sleep(3)
+    max_attempts = 30
+    for attempt in range(max_attempts):
```

Ruff (0.12.2)

Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)